Platform/QemuBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -212,10 +212,10 @@ BoardInit (
   switch (InitPhase) {
   case PreSiliconInit:
     GpioInit ();
-    SpiConstructor ();
     EnableLegacyRegions ();
     VariableConstructor (PcdGet32 (PcdVariableRegionBase), PcdGet32 (PcdVariableRegionSize));
     if (!FeaturePcdGet (PcdStage1BXip)) {
+      SpiConstructor ();

```

Comment:
I think we can just skip variable test for QSP platform.  
And variable depends on SPI constructor.  Need to check if your VariableConstructor will work or not after moving.

Reply:
Done.